port/scorecard/resource_test.go

```diff
@@ -0,0 +1,299 @@
+
+	var testAccActionConfigUpdate = testAccCreateBlueprintConfig(blueprintIdentifier) + fmt.Sprintf(`
+	resource "port_scorecard" "test" {
+		identifier = "%s"
+		title      = "Scorecard 2"
+		blueprint  = "%s"
+		rules = [{
+					identifier = "hasTeam"
+					title      = "Has Team"
+					level      = "Bronze"
+					query = {
+						combinator = "or"
+						conditions = [{
+							property = "$team"
+							operator = "isNotEmpty"
+						}]
+					}
+				}]
+		depends_on = [
+		port_blueprint.microservice
+		]
+	 }`, scorecardIdentifier, blueprintIdentifier)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.TestAccPreCheck(t) },
+		ProtoV6ProviderFactories: acctest.TestAccProtoV6ProviderFactories,
+
+		Steps: []resource.TestStep{
+			{
+				Config: acctest.ProviderConfig + testAccActionConfigCreate,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("port_scorecard.test", "title", "Scorecard 1"),
+					resource.TestCheckResourceAttr("port_scorecard.test", "blueprint", blueprintIdentifier),
+					resource.TestCheckResourceAttr("port_scorecard.test", "rules.#", "1"),
+					resource.TestCheckResourceAttr("port_scorecard.test", "rules.0.identifier", "hasTeam"),
+					resource.TestCheckResourceAttr("port_scorecard.test", "rules.0.title", "Has Team"),
+					resource.TestCheckResourceAttr("port_scorecard.test", "rules.0.level", "Gold"),
+					resource.TestCheckResourceAttr("port_scorecard.test", "rules.0.query.combinator", "and"),
+					resource.TestCheckResourceAttr("port_scorecard.test", "rules.0.query.conditions.#", "1"),
+					resource.TestCheckResourceAttr("port_scorecard.test", "rules.0.query.conditions.0.property", "$team"),
+					resource.TestCheckResourceAttr("port_scorecard.test", "rules.0.query.conditions.0.operator", "isNotEmpty"),
+				),
+			},
+			{
+				Config: acctest.ProviderConfig + testAccActionConfigUpdate,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("port_scorecard.test", "title", "Scorecard 2"),
+					resource.TestCheckResourceAttr("port_scorecard.test", "blueprint", blueprintIdentifier),
+					resource.TestCheckResourceAttr("port_scorecard.test", "rules.#", "1"),
+					resource.TestCheckResourceAttr("port_scorecard.test", "rules.0.identifier", "hasTeam"),
+					resource.TestCheckResourceAttr("port_scorecard.test", "rules.0.title", "Has Team"),
+					resource.TestCheckResourceAttr("port_scorecard.test", "rules.0.level", "Bronze"),
+					resource.TestCheckResourceAttr("port_scorecard.test", "rules.0.query.combinator", "or"),
+					resource.TestCheckResourceAttr("port_scorecard.test", "rules.0.query.conditions.#", "1"),
+					resource.TestCheckResourceAttr("port_scorecard.test", "rules.0.query.conditions.0.property", "$team"),
+					resource.TestCheckResourceAttr("port_scorecard.test", "rules.0.query.conditions.0.operator", "isNotEmpty"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccPortScorecardImport(t *testing.T) {
+
```

Comment:
remove blank line

its also a good idea to add an auto linter and a git action to test it before approving merges

Reply:
There is a auto linter actually, but probably go doesn't care about 1 line